test/XRP/fakes/fake-xrp-protobufs.ts

```diff
@@ -455,7 +466,11 @@ export {
   testIsValidated,
   testLedgerIndex,
   testCurrencyProto,
-  testPathElementProto,
+  testAccountAddressIssuer,
+  testAccountPathElementProto,
+  testAccountCurrencyPathElementProto,
+  testAccountIssuerPathElementProto,
+  testCurrencyIssuerPathElementProto,
```

Comment:
typically group all invalid protos in order, in second half of export list, and valid ones in order in first half